tools/oeedger8r/README.md

```diff
@@ -71,10 +71,17 @@ improved plugin model as it is a copy of Intel's code.
 
 ### Best Practices
 
-We use [ocamlformat](https://github.com/ocaml-ppx/ocamlformat) to format our
-code (such as `Emitter.ml`, but not Intel's code). It is the final say in
-formatting. This should be setup to run automatically in one's editor, as it has
-not yet been setup in CI.
+We use [ocamlformat v0.12](https://github.com/ocaml-ppx/ocamlformat) (bundled
```

Comment:
- Is ocamlformat run as part of commit-checks?

Reply:
Not yet. I was looking to do that, but I'd want to filter it such that it's only run if the OCaml files are touched, which means more extensive changes to the hook than I expected.